package.json

```diff
@@ -85,6 +85,7 @@
     "@atlaskit/navigation": "33.3.8",
     "@atlaskit/onboarding": "6.1.14",
     "@atlaskit/page": "8.0.12",
+    "@atlaskit/panel": "^0.3.5",
```

Comment:
Can you please pin the depenency to the exact version?

Reply:
Fixed sorry for this.